tcms_django_plugin/result.py

```diff
@@ -93,12 +93,13 @@ def stopTestRun(self):
 
 class DebugSQLTestResult(TestResult):
     def __init__(self, stream, descriptions, verbosity, **kwargs):
+        super().__init__(stream=stream, descriptions=descriptions,
+                         verbosity=2, **kwargs)
```

Comment:
The signatures between the two classes should be the same. Pylint used to complain when inherited methods have different signatures even though in this case they are almost equivalent.

For example here verbosity is not a default parameter but in the inherited class it is.

Then when calling super() all arguments should be passed directly without any changes, e.g. `verbosity=verbosity`.